bin/ream/src/cli/mod.rs

```diff
@@ -32,6 +32,19 @@ pub struct NodeConfig {
         value_parser = network_parser
     )]
     pub network: Arc<NetworkSpec>,
+
+    #[arg(
+        long,
+        help = "The directory for storing application data. If used together with --ephemeral, new child directory will be created."
+    )]
+    pub data_dir: Option<PathBuf>,
```

Comment:
Would a default value be useful here? Like `~/.ream` ?

Reply:
We don't need to set a default here as `setup_data_dir() will use the default application data directory  for the specific OS